Pipfile

```diff
@@ -8,14 +8,15 @@ requests = "*"
 
 [dev-packages]
 "flake8" = "*"
-coverage = "*"
-coveralls = "*"
+coverage = "5.5"
+coveralls = "2.2.0"
 bumpversion = "*"
 setuptools = "*"
 twine = "*"
-pytest = "*"
-pytest-cov = "*"
+pytest = "6.2.1"
+pytest-cov = "2.11.1"
```

Comment:
We shouldn't need to pin the package versions like this in the Pipfile.

Reply:
This has been changed to not use exact versions.